netutils/dhcpc/Kconfig

```diff
@@ -29,4 +29,12 @@ config NETUTILS_DHCPC_RETRIES
 		This setting determines how many times resolver retries request
 		until failing.
 
+config NETUTILS_DHCPC_BOOTP_FLAGS
+	hex "Flags of Bootstrap"
+	default 0x0
```

Comment:
Should the default be 0x8000 to retain broadcast as the default instead of unicast?

Reply:
Got